src/std/make.ss

```diff
@@ -757,6 +765,22 @@ TODO:
      (compile-exe-gsc-opts rest))
     (else opts)))
 
+(def (compile-exe/context mod opts settings)
+  (cond
+   ((or (settings-build-release settings)
+        (getenv "GERBUIL_BUILD_RELEASE" #f))
```

Comment:
GERBUIL ???

That smells like one pun too much.

Reply:
lol, fixing

Reply:
fixed.